### PR TITLE
Responsive: Make my personal portfolio a responsive website

### PR DESCRIPTION
**Pull request by kemigabocatherine**
### In this PR;

**There is the use of media queries to;**
- design a responsive website for mobile and desktop devices
- Added media query and CSS styles for the desktop version of the portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,316 +15,226 @@
...
+          <header class="toolbar">
+              <a><img class="logo" src="images/LogoPlaceholder.png" alt="My Logo"/></a>
+              <nav><img class="menu" src="images/hamburger.png" alt="menu"/>
+                  <ul class="dmenu">
+                      <li class="port"><a href="#home">Portfolio</a></li>
+                      <li class="ab"><a href="#about">About</a></li>
+                      <li class="cont"><a href="#contact">Contact</a></li>
+                  </ul>
+              </nav>    
+          </header>
```
Comment (mahbubul14) [inline index.html]: - [ ] For readability, please add two spaces of indentation. Do not use the tab key. You did it throughout your code in the `HTML` file. Please fix others also. For the HTML & CSS best practices we should follow the rules. :writing_hand:

File: index.html [hunk 2]
```diff
@@ -15,316 +15,226 @@
...
+      <!--Intoduction-->
+      <section class="home" id="home">
+          <header class="toolbar">
+              <a><img class="logo" src="images/LogoPlaceholder.png" alt="My Logo"/></a>
+              <nav><img class="menu" src="images/hamburger.png" alt="menu"/>
+                  <ul class="dmenu">
+                      <li class="port"><a href="#home">Portfolio</a></li>
+                      <li class="ab"><a href="#about">About</a></li>
+                      <li class="cont"><a href="#contact">Contact</a></li>
+                  </ul>
```
Comment (mahbubul14) [inline index.html]: - [ ] Kindly fix your navigation item for desktop view :desktop_computer: . It doesn't match the Figma design. Your page looks like now:
n1

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
# Required Changes ♻️

Hi @kemigabocatherine , 

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

### Mandatory Changes

- [ ] Kindly add a live demo link of the project in your `readme` file.

- [ ] As per your project requirements, you should use Grid for at least one of the layout of the elements.

- [ ] Kindly check the comments under the review to see more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: Correct Github flow.
:heavy_check_mark: Descriptive title and summary for the PR.

### Optional Suggestions

- [ ] In the contact section of your readme file, It would be better if you use Author instead of Contact.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by mke2111** [comment]
Project Under Second Opinion Review.
Kindly wait for a response. 😄

**Review by mke2111 — COMMENTED** [review]
Hello @kemigabocatherine  👋,

_This is an additional review as a result of the second opinion request._

> I will try to make this clarification and I hope by the end we shall all have the same understanding of the request.

It is a best practice to have an indentation of 2 spaces for HTML as well. This does not only apply to CSS. As mentioned in the [HTML & CSS best practices article](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), specifically, option 3. You can set up your favorite code editor to have a default spacing of 2 even when you use the `tab` key.

Make these requested changes and submit another CRR whenever you are ready.

Keep rocking 🚀!

**Review by BenMukebo — APPROVED** [review]
Hi @kemigabocatherine , 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**